### PR TITLE
Update all get/set state functions to support string key argument

### PR DESCRIPTION
**Pull request by ratik21**
#193 to be merged first

### CODE REVIEW

File: packages/algorand-js/src/lib/parsing.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export function assertOnlyDigits (val: string): void {
 }
 
 /**
- * Description: assert words length
+ * assert words length
```
Comment (robert-zaremba) [inline packages/algorand-js/src/lib/parsing.ts:20]:
```suggestion
 * assert that a line has given number of words
```

File: packages/algorand-js/src/lib/parsing.ts [hunk 2]
```diff
@@ -86,7 +94,7 @@ export function convertToBuffer (s: string, encoding?: EncodingType): Buffer {
 }
 
 /**
- * Description: Returns base64 or base32 string
+ * Returns base64 or base32 string
```
Comment (robert-zaremba) [inline packages/algorand-js/src/lib/parsing.ts:97]: Please update the the description about the `args` argument and describe when we convert to base64 and when to base32

File: packages/algorand-js/src/lib/parsing.ts [hunk 3]
```diff
@@ -110,7 +118,7 @@ function base64OrBase32 (args: string[], line: number): [string, EncodingType] {
 }
 
 /**
- * Description: returns encodingtype and string from words list
+ * returns encodingtype and string from words list
```
Comment (robert-zaremba) [inline packages/algorand-js/src/lib/parsing.ts:121]: Please update the description to make it more intuitive. what string are we returning and what is an encodingtype.

**Review by robert-zaremba — APPROVED** [review]
looks good. left few comments to update function docs.

File: packages/algorand-js/src/lib/parsing.ts (hunk 2 above)
Comment (ratik21) [reply packages/algorand-js/src/lib/parsing.ts:97]: ok

File: packages/algorand-js/src/lib/parsing.ts (hunk 3 above)
Comment (ratik21) [reply packages/algorand-js/src/lib/parsing.ts:121]: yes, ok

**Pull request merged**